src/components/Orders/Orders.js

```diff
@@ -1,25 +1,23 @@
-import React from 'react';
-import './Orders.css';
+import React from "react";
+import "./Orders.css";
 
-const Orders = props => {
-  const orderEls = props.orders.map(order => {
+const Orders = (props) => {
```

Comment:
I think my preference for a final assessment would be that we consistently use function declarations rather than a mix of the two.  But I don't know that it really matters all that much.

Reply:
We have two options! One is to keep things the way they are and add extra credit for code style. Alternatively, we can provide all functions as function descriptions and hope that students will follow the same style.